src/src/ControllerQueue/C015_queue_element.cpp

```diff
@@ -6,6 +6,17 @@
 
 C015_queue_element::C015_queue_element() {}
 
+C015_queue_element::C015_queue_element(C015_queue_element&& other)
+  : idx(other.idx), _timestamp(other._timestamp), TaskIndex(other.TaskIndex)
+  , controller_idx(other.controller_idx), valuesSent(other.valuesSent)
+  , valueCount(other.valueCount)
+{
+  for (byte i = 0; i < VARS_PER_TASK; ++i) {
+    txt[i]  = std::move(other.txt[i]);
+    vPin[i] = other.vPin[i];
+  }
```

Comment:
ref. above, this also can be omitted in case `txt = other.txt;` could work, consider `std::array<String, VARS_PER_TASK>;`? or a custom object implementing `Object& operator=(Object&&) noexcept;'